src/Models/ErrorViewModel.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+
+namespace TrainingNet.Models
+{
+    public class ErrorViewModel
+    {
+        public string RequestId { get; set; }
+
+        public bool ShowRequestId => !string.IsNullOrEmpty(RequestId);
+    }
+}
```

Comment:
Add an empty line at EOF (For some reason, github doesnt like it when files don't end on an empty line)

Reply:
![image](https://user-images.githubusercontent.com/16925524/45383897-db394500-b5e2-11e8-8754-3801b365bfb5.png)